Parser/asdl_c.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env tauthon
```

Comment:
We need to change this one back, actually, (it's part of the build process; python needs python to build itself). This is why the build is breaking on OSX and Linux, (https://travis-ci.org/naftaliharris/tauthon/jobs/205960243).

Let's restrict these changes to Demo/*, Demo/*, Lib/*, and Tools/* I think. The other files may be needed in the build process for different platforms.